neon_utils/default_configurations/default_user_conf.yml

```diff
@@ -48,3 +48,7 @@ location:
   country: United States
   tz: America/Los_Angeles
   utc: '-8.0'
+
+util_params:
```

Comment:
Not sure I know what else falls under `util_params` from a user's perspective. I was thinking `skills` makes some sense if this is where other per-user settings for the skill manager or intent service go, but not sure how this may expand in the future.. No requirement to change this, just a note for consideration.

Reply:
I agree. I thought we should not use `interface' and 'skills' because that would mirror the local config